packages/browser-client/src/Components/PeerButtons.tsx

```diff
@@ -421,16 +238,16 @@ export default function PeerButtons() {
                 width={'33%'}
                 title="Add content to offer"
                 onClick={() => {
-                  addToOffer('block')
+                  peerActions.addToOffer('block')
```

Comment:
Does it make sense to have this "Offer Block" since the idea of these buttons is to allow direct control of the wire methods of the protocol?  We don't have a concept of offering a "block" since a block isn't a discrete piece of content in the portal network.  If we remove Find/Offer Block, we can greatly simplify the logic of the PeerReducer and I dunno if it really gains us anything.

Reply:
Yeah I suppose it does diverge from the point of the peer controls, since it does a bunch of extra logic and isn't pure Wire Protocol.